source/develop/release-management/features/storage/4k-blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 4k block device support for the storage domains
+category: feature
+authors: dchaplyg
+feature_name: 4k block device support for the storage domains
+feature_modules: vdsm
+feature_status: Planning
+---
+
+# 4k block device support for the storage domains
+
+# Summary
+
+VDSM assumes, that 512b is a minimally accessible block on the storage device, which can be written or read atomically. That was true for a long time, but now most of the storage devices changed their block size to the 4096 bytes (4 kilobytes). It is still compatible with VDSM, case while we are requesting 512 atomic read or write, underlying storage will atomically read 4096 bytes, throw away 3584 bytes and return just 512. As it still acts same way, it has no impact on the VDSM. 
```

Comment:
512b -> 512 bytes

 You need to backup this claim some reference:
    
    During virt-v2v, we can use qemu-img measure on the overlay. It gives a conservative
    estimate that will never fail, based on the number of allocated blocks and additional
    metadata required for the destination file format.

Reply:
512b fixed

I mostly talking on a 512 emulation details, not performance.